apidoc/Titanium/Calendar/Attendee.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Titanium.Calendar.Attendee
+summary: An object that represents a single attendee of an event.
+description: |
+    The API supports retrieving information about the attendee of an event.  
+
+extends: Titanium.Proxy
+platforms: [iphone, ipad]
+since: "6.0.0"
+createable: false
+
+properties:
+  - name: isOrganiser
+    summary: Indicates whether this attendee is the event organiser.
+    type: Boolean
+    platforms: [iphone, ipad]
+
+  - name: name
+    summary: The attendee name.
+    type: String
+    platforms: [iphone, ipad]
+
+  - name: email
+    summary: The attendee email.
+    description: |
+        Due to privacy concerns in iOS this value will be empty if the attendee is the device owner. 
```

Comment:
Why is it empty? I can't believe that there is no chance to receive it. And if not, we need to remove the property since it cannot be used.